src/ec/packages/ec-component-ecl-compliance/components/ecl-compliance-gallery.html.twig

```diff
@@ -0,0 +1,82 @@
+{#
+  Parameters:
+  - "items" (array) (default: [])
+#}
+
+{% extends "@ecl-twig/ec-component-ecl-compliance/ecl-compliance.html.twig" %}
+{% set component = 'gallery' %}
+{% set message = '' %}
+
+{% set less_than_five_case = false %}
+{% set no_video_sources_case = false %}
+{% set no_icon_case = false %}
+{% set no_image_source_case = false %}
+{% set no_item_alt_case = false %}
+{% set no_video_sources_case = false %}
+{% set no_video_source_case = false %}
+
+{% set no_items_case = items is empty or items is not iterable %}
+{% if no_items_case == false %}
+  {% set less_than_five_case = items|length < 5 %}
+  {% for item in items %}
+    {% if (item.video is defined and item.video is not empty) %}
+      {% if item.icon is not defined or item.icon is empty %}
+        {% set no_icon_case = true %}
+      {% endif %}
+      {% if item.video.sources is defined and
+        item.video.sources is not empty and
+        item.video.sources is iterable %}
+        {% for source in item.video.sources %}
+          {% if (source.src is not defined or source.src is empty) or
+            (source.type is not defined or source.type is empty) %}
+            {% set no_video_source_case = true %}
+          {% endif %}
+        {% endfor %}
+      {% else %}
+        {% set no_video_sources_case = true %}
+      {% endif %}
+    {% elseif (item.image is defined and item.image is not empty) %}
+      {% if (item.image.src is not defined or item.image.src is empty) and
+        (item.path is not defined or item.path is empty) %}
+        {% set no_image_source_case = true %}
+      {% endif %}
+      {% if (item.image.alt is not defined or item.image.alt is empty) and
+        (item.alt is not defined or item.alt is empty) %}
+        {% set no_item_alt_case = true %}
+      {% endif %}
+    {% elseif item.path is not defined or item.path is empty %}
+      {% set no_image_source_case = true %}
+    {% endif %}
+  {% endfor %}
+{% endif %}
+
+{% set not_compliant =
+  no_items_case or
+  less_than_five_case or
+  no_image_source_case or
+  no_video_sources_cases or
+  no_video_source_case or
+  no_icon_case or
+  no_item_alt_case %}
+
+{% if no_items_case %}
+  {% set message = '"items" needs to be an array containing at least one "gallery item".' %}
+{% endif %}
+{% if less_than_five_case %}
+  {% set message = 'You have less then five items in your gallery, in this case you should use use an unordered list or a media container.' %}
```

Comment:
This is the only personalized message with "you have" which feels strange comparing with the others which are impersonal form.